pallets/providers/src/utils.rs

```diff
@@ -1990,6 +2099,14 @@ impl<T: pallet::Config> ReadProvidersInterface for pallet::Pallet<T> {
             || MainStorageProviders::<T>::contains_key(&who)
             || Buckets::<T>::contains_key(&who)
     }
+
+    fn is_provider_insolvent(who: Self::ProviderId) -> bool {
+        InsolventProviders::<T>::get(&who).is_some()
+    }
+
+    fn insolvency_block(who: Self::ProviderId) -> Option<TickNumberFor<T>> {
+        InsolventProviders::<T>::get(&who)
+    }
```

Comment:
Because the type is generic. Tick number can be anything. 

But I do agree with your suggestion to the nomenclature. Renamed all instances of `insolvency_block` to `insolvency_tick`.